src/v3/test/integration/__snapshots__/authenticator-enroll-google-authenticator.test.tsx.snap

```diff
@@ -495,10 +495,11 @@ exports[`authenticator-enroll-google-authenticator renders form renders challeng
                       >
                         <input
                           aria-invalid="false"
-                          autocomplete="one-time-code"
+                          autocomplete="off"
```

Comment:
These snapshots show one branch of the conditional. Could you add tests to check that `autocomplete` and `inputmode` are set correctly if `isAndroidOrIos` is mocked to return true (i.e., "simulate" the mobile scenario)? Otherwise, the conditional behavior isn't unit tested AFAICT.